docs/content/en/docs/references/api/grpc.md

```diff
@@ -1010,8 +1012,13 @@ For Cancelled Error code, use range 800 to 850.<br>
 | CONFIG_MULTI_IMPORT_PROFILE_CONFLICT_ERR | 1211 | Same config imported at least twice with different set of profiles |
 | CONFIG_PROFILES_NOT_FOUND_ERR | 1212 | Profile selection did not match known profile names |
 | CONFIG_UNKNOWN_API_VERSION_ERR | 1213 | Config API version not found |
+| CONFIG_UNKNOWN_VALIDATOR | 1214 | The validator is not supported in skaffold-managed mode (not whitelisted). |
+| CONFIG_UNKNOWN_MUTATOR | 1215 | The mutator is not supported in skaffold-managed mode (not whitelisted). |
 | INSPECT_UNKNOWN_ERR | 1301 | Catch-all `skaffold inspect` command error |
 | INSPECT_BUILD_ENV_ALREADY_EXISTS_ERR | 1302 | Trying to add new build environment that already exists |
+| KPTFILE_INVALID_YAML_ERR | 1401 | Kptfile errors The Kptfile is not a valid yaml file |
+| KPTFILE_INVALID_SCHEMA_ERR | 1402 | The Kptfile is not a valid API schema |
+| KPTFILE_INIT_ERR | 1403 | The Kptfile cannot be created via `kpt pkg init` or `kpt live init`. |
```

Comment:
Please prefix these with "DEPLOY_" as they belong to "Deploy" phase

Reply:
The kptfile init happens in both render (in most cases) and deploy (if users decides to use "skip-render" flag). I was thinking of using this error in both phases. Or do you suggest we separate two errors, one is DEPLOY_KPTFILE_INIT_ERR and other is RENDER_KPTFILE_INIT_ERR?

Reply:
I see the prefix comment. Defined both DEPLOY_KPTFILE_INIT_ERR and RENDER_KPTFILE_INIT_ERR